torch/_inductor/codegen/mps.py

```diff
@@ -978,7 +1045,7 @@ def define_kernel(
             if V.graph.cpp_wrapper:
                 kernel_name = f"{mps_lib_name}_func"
             else:
-                kernel_name = f"{mps_lib_name}.generated_kernel"
+                kernel_name = f"{mps_lib_name}"
```

Comment:
Hmm, how this is related to dynamic shapes?

Reply:
Ah yeah! So when generating the dynamic input variables there's some logic in wrapper to do `f"{kernel_name}_xnumel"`. Previously the kernel_name would be like `mps_lib_0.generated_kernel` which would result in the variable name "mps_lib_0.generated_kernel_xnumel", which is an invalid name because of the period. So I changed the kernel_name to just be "mps_lib_0" so that the variable names would be "mps_lib_0_xnumel"